phpunit/tests/fonts/font-library/wpRestFontCollectionController/wpRestFontCollectionsController.php

```diff
@@ -0,0 +1,300 @@
+<?php
+/**
+ * Unit tests covering WP_REST_Font_Collections_Controller functionality.
+ *
+ * @package WordPress
+ * @subpackage REST API
+ *
+ * @group restapi
+ */
+class Tests_Fonts_WpRestFontCollectionsController extends WP_Test_REST_Controller_Testcase {
```

Comment:
I'd expect a class name similar to `WP_Test_REST_Posts_Controller`. Is there a reason you're using camelcase here?

Reply:
Renamed.  I think it's closer to what's expected now.